components/NavigationDrawer/src/MDCBottomDrawerPresentationController.m

```diff
@@ -78,6 +78,12 @@ - (void)presentationTransitionWillBegin {
   }
   bottomDrawerContainerViewController.shouldIncludeSafeAreaInContentHeight =
       self.shouldIncludeSafeAreaInContentHeight;
+  if (self.elevation > 0) {
```

Comment:
Based on the setter below, all elevation values should be passed-through to the container VC.

Reply:
Removed checks.